src/proto/workload.rs

```diff
@@ -0,0 +1,2283 @@
+// This file is generated by rust-protobuf 2.23.0. Do not edit
```

Comment:
Since we generate it here, maybe it's not needed to have a build file? Would save some compilation time. We can re-generate it manually when the Workload API gets updated.

Reply:
I removed the build file.